README.adoc

```diff
@@ -20,24 +20,31 @@ Sample apps that demonstrate out-of-box websockets listener based peer-to-peer d
 
 == Introduction
 
-Couchbase Lite 2.8 release supports out-of-the-box support for secure link:https://docs.couchbase.com/couchbase-lite/2.8/swift/learn/swift-landing-p2psync.html[Peer-to-Peer Sync], over websockets directly between Couchbase Lite enabled clients in IP-based networks without the need for a centralized control point (i.e. no Sync Gateway or Couchbase server required)
+Couchbase Lite supports out-of-the-box support for secure
+https://docs.couchbase.com/couchbase-lite/current/swift/learn/swift-landing-p2psync.html[Peer-to-Peer Sync]
+, over websockets directly between Couchbase Lite enabled clients in IP-based networks without the need for a centralized control point (i.e. no Sync Gateway or Couchbase server required)
 
 
 == iOS Inventory App Tutorial in Swift
 === Prerequisites
 This tutorial assumes familiarity with building swift apps with Xcode and with Couchbase Lite.
 
-* If you are unfamiliar with the basics of Couchbase Lite, it is recommended that you follow the link:https://docs.couchbase.com/couchbase-lite/2.7/swift.html[Getting Started] guides
+* If you are unfamiliar with the basics of Couchbase Lite, it is recommended that you follow the
+https://docs.couchbase.com/couchbase-lite/2./swift.html[Getting Started]
```

Comment:
@ibsoln  The link does not seem correct

Reply:
Fixed here: https://github.com/couchbaselabs/couchbase-lite-peer-to-peer-sync-examples/pull/14/commits/da97d325e60107a869c627431f0d268981f24588